src/consumer/consumer.py

```diff
@@ -100,11 +100,39 @@ def heartbeat_callback(ch, method, properties, body):
             services_last_timestamp[json_data["service"]] = json_data["timestamp"]
         except Exception as e:
             print(f"\33[31mError indexing to Elasticsearch: {e}\33[0m")
+    
+ 
+    def send_error_email(service, timestamp, error, status):
```

Comment:
Men moet dus eerst wachten dat mailing kan consumer om te testen (enkel mergen in de main wanneer het effectief werkt)  
Ik mag vermoeden dat je hun xml & queue respecteerd?

Reply:
Voor de rest lijkt het nu redelijk goed, dan moet je zien met de mailing team waneer zij kunnen consumen en dan testen met hun systeem.

Ook: als nu iemand down is gaat het om de 5s een mail sturen, zou je niet maar 1 keer een mail sturen per keer da ze up=>down gaan?